### ISSUE TITLE
Improve sidebar opening performance

### ISSUE THREAD

**Issue by Croydon** [Croydon/vertical-tabs-reloaded#158]
It's terrible right now.

https://bugzilla.mozilla.org/show_bug.cgi?id=1371255

**Comment by Luckz** [comment]
Several minutes with 5000 tabs ;p

**Comment by Croydon** [comment]
0.11.0 should be much, much better.

Todo: Run actual performance tests.

**Comment by Croydon** [comment]
So I did run some tests and I'm really, really happy 😄 
The refactoring did pay off big time.

![vtr-performance-issue-158-part-1](https://user-images.githubusercontent.com/1593194/40571595-fe09f940-609b-11e8-8e29-525c0191cae4.PNG)
![vtr-performance-issue-158-part-2](https://user-images.githubusercontent.com/1593194/40571597-ff8b276c-609b-11e8-9d7d-13e921797d54.PNG)

Tested with Firefox Nightly 62.0a1 on Windows 10. Absolute numbers obviously differ from system to system, but the overall improvement should pay off on every system.

Fun fact: The test with **2.000** loaded tabs was very, very painful. Including several crashes.
